src/main/lombok/org/heigit/ohsome/ohsomeapi/controller/DefaultSwaggerParameters.java

```diff
@@ -6,12 +6,10 @@
   public static final String BBOX = "8.625,49.3711,8.7334,49.4397";
   public static final String HIGHWAY_KEY = "highway";
   public static final String BUILDING_KEY = "building";
-  public static final String HOUSENUMBER_KEY = "addr:housenumber";
-  public static final String RESIDENTIAL_VALUE = "residential";
   public static final String TYPE_FILTER = "type:way";
   public static final String HIGHWAY_FILTER = "highway=residential";
   public static final String BUILDING_FILTER = "building=*";
-  public static final String HOUSENUMBER_FILTER = "type:node and \"addr:housenumber=*\"";
+  public static final String HOUSENUMBER_FILTER = "type:node and \"addr:housenumber\"=*";
```

Comment:
Would you recommend writing the key in that way using the quotation marks?

I think that tag here solely had those marks, because me (or whoever added that in the past) thought, that they are required for this specific key when using the filter parameter. Otherwise, we can just write it without the quotation marks and follow the conventions of the docs as you described it in the comment before.

Reply:
> Would you recommend writing the key in that way using the quotation marks?

I would normally write it without the quote marks, but there's also nothing wrong with using them.

I left them in this PR because I thought it might be a hint for new users playing around in the swagger UI so find out that one can also define tag keys with `"`s.